xonsh/ptk/completer.py

```diff
@@ -32,5 +36,25 @@ def get_completions(self, document, complete_event):
                                                      begidx,
                                                      endidx,
                                                      self.ctx)
+            if len(completions) <= 1:
+                pass
+            elif len(os.path.commonprefix(completions)) == len(prefix):
```

Comment:
Is there a reason for this check, and not always to do this?  This won't work with the current completer, where not all completions start with the same length substring (some are wrapped in quotes, for example).